README.rst

```diff
@@ -2,93 +2,103 @@
 ``sunpy``
 *********
 
-|Latest Version| |DOI| |repostatus| |python|
-|ci| |codecov| |Docs|
+|Latest PyPi Version|
+|Latest Conda Version|
+|DOI|
+|repostatus|
+|python|
+|ci|
+|codecov|
+|Docs|
 |matrix|
 |Powered by NumFOCUS|
```

Comment:
This is a bit tangential, but our number of badges continues to increase. Should we organize them into a table?

Reply:
I will never forgive you.